plugins/pulp_rpm/plugins/importers/yum/utils.py

```diff
@@ -130,6 +130,8 @@ def fake_xml_element(repodata_snippet):
         # best second guess we have, and it will never fail due to the nature
         # of the encoding.
         codec = 'ISO-8859-1'
+    except UnicodeDecodeError:
+        repodata_snippet = repodata_snippet.decode("ISO-8859-1")
```

Comment:
To make a quick hack we can try to decode as 'UTF-8' here and I believe in most cases it will work.
The main issue is that for some reason the data on input can be either in str or unicode forms.